src/components/Collapsible/index.jsx

```diff
@@ -35,10 +35,9 @@ const CollapseButton = ({
   handleClick, collapsed, strings, buttonStyle, handleHoverOn, handleHoverOff,
 }) => (
   <ButtonContainer style={buttonStyle} onClick={handleClick} onMouseEnter={handleHoverOn} onMouseLeave={handleHoverOff}>
-    {collapsed ?
-    [<span>{strings.general_show}</span>, <IconPlusSquare />]
-    :
-    [<span>{strings.general_hide}</span>, <IconMinusSquare />]}
+    {collapsed
+      ? [<span>{strings.general_show}</span>, <IconPlusSquare />]
+      : [<span>{strings.general_hide}</span>, <IconMinusSquare />]}
```

Comment:
**This is absolutely out of scope of this pr**, but I think we should add explicit keys to the elements in array 

```
Warning: Each child in a list should have a unique "key" prop.

Check the render method of `App`. See https://fb.me/react-warning-keys for more information.
    in p (at src/index.js:9)
    in App (at src/index.js:17)
```

Reply:
Completely agree. These warnings have been driving me insane